app/models/papi_client.rb

```diff
@@ -316,13 +316,6 @@ def get_command_line_options(study_file, action)
         taxon = study_file.taxon
         opts += ["--taxon-name", "#{taxon.scientific_name}", "--taxon-common-name", "#{taxon.common_name}",
                  "--ncbi-taxid", "#{taxon.ncbi_taxid}"]
-        if taxon.current_assembly.present?
-          assembly = taxon.current_assembly
-          opts += ["--genome-assembly-accession", "#{assembly.accession}"]
-          if assembly.current_annotation.present?
-            opts += ["--genome-annotation", "#{assembly.current_annotation.name}"]
```

Comment:
I see these parameters exist, but are now longer used in Ingest Pipeline.  

Could you refresh my memory on how genome assembly and genome annotation will be associated with gene expression matrices that are processed by Ingest Pipeline?

Reply:
They're associated at the `StudyFile` level, which happens when the matrix is created/updated.  We only ask for the species, but there are defaults that kick in that will return an assembly & annotation if we ask for them at the file level.

To my knowledge, I don't think we use the assembly or annotation at the `Gene` level anywhere, but if we did or want to, it's easy enough to get from the parent file.